app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java

```diff
@@ -410,9 +397,6 @@ public void onItemClicked(Presenter.ViewHolder itemViewHolder, Object item,
                         Intent settings = new Intent(getActivity(), SettingsActivity.class);
                         getActivity().startActivity(settings);
                         break;
-                    case REPORT:
-                        Utils.reportError(getActivity(), "Send Log to Dev");
```

Comment:
Should this function also be removed?

Reply:
I'm fine with it either way. That method just handles displaying a modal confirmation message, so we may reuse it.

Reply:
I left it in as it's referenced elsewhere - I'm happy to remove it though.